docs/docs/Installation/index.md

```diff
@@ -11,3 +11,20 @@ npm install --save @walletconnect/solib
 ```bash
 yarn add @walletconnect/solib
 ```
+
+### Note on QRCodes and Web3Modal's standalone modal
+If you plan on setting `qrcode: false` and handling QRCode display yourself when
+using `WalletConnectConnector`, be sure to mark the web3modal dependencies as external in the rollup or webpack config.
+They are peer dependencies and are not downloaded automatically, and need to be
+marked as such to avoid import analysis tools false flagging them.
+
+Example in vite
+```json
+export default defineConfig({
+  rollupOptions: {
+    external: ["@web3modal/core", "@web3modal/ui"]
```

Comment:
Just double checking, are these (external) guides still needed, after our convo in slack?

Reply:
Yes, it's needed for this. Might be a vite specific thing, but works with plain rollup for some reason